jenkins/install-deps.sh

```diff
@@ -1,8 +1,46 @@
 #!/bin/bash -xue
 
-sudo add-apt-repository --yes ppa:fkrull/deadsnakes
-sudo apt-get update
-sudo apt-get install --yes python2.6 python2.6-dev
+function install_deb(){
+    sudo aptitude install -y python-dev libffi-dev python-pip    
+}
 
-sudo aptitude install -y python-dev libffi-dev python-pip
-sudo pip install tox
+function install_centos(){
+    sudo yum -y install python-devel libffi-devel
+    # pip is not in the standard repo
+    sudo yum -y install epel-release
+    sudo yum -y install python-pip
+
+    # This gets installed automatically in debian because recommended packages
+    # Doing that manually in centos.
+    sudo yum -y install gcc
+
+}
+
+
+function is_centos(){
+    [[ -f /etc/centos-release ]]
+}
+
+function is_deb(){
+    [[ -f /etc/debian_version ]]
+}
+
+function pip_install(){
```

Comment:
This function is mis named.

It should be install_tox() or pip_install_tox(). But no need to make a function for one line of code. You should inline the code